package.json

```diff
@@ -27,6 +28,7 @@
     "request-promise": "^4.2.4"
```

Comment:
Can you replace `request-promise` with `axios`? Open a new separate PR if needed.

Reply:
Will replace `request-promise` with `axios` as part of this [snyk](https://cloudelements.atlassian.net/browse/ENG-1633) ticket